index.html

```diff
@@ -80,7 +80,109 @@ <h1 class="mn__title">
+         </div>
+        </div>
+        <div class="proj__container" id="proj__3">
+        <img class="proj__img" src="img/Work 3.webp" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__4">
+        <img class="proj__img" src="img/Work 4.jpg" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__5">
+        <img class="proj__img" src="img/Work 5.webp" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__6">
+        <img class="proj__img" src="img/Work 6.jpg" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+      </section>
+      <section class="abt__me">
+        <div class="abt__me__title"><h2>About me</h2></div>
+         <div id="abt__txt">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, <br> don't hestiate to contact me.
+          </p>
+        </div>
+        <button class="see__proj__btn">Get My Resume</button>
+      </section>
+      <article class="lang__used"></article>
```

Comment:
- [ ] While you did an amazing job creating 'About me' section you should also add the required cards at the end as you can see in the image below. Also please don't forget to use flexbox for it :+1: 

![Screenshot from 2023-01-12 19-34-02](https://user-images.githubusercontent.com/105117832/212210210-80201609-1937-4632-9165-75ff7c76246c.png)